.github/workflows/release_workflows.yml

```diff
@@ -9,7 +9,7 @@ on:
         description: major, minor, or patch
         default: patch
       release_notes:
-        description: A text message summarizing the changes in this release
+        description: A text message summarizing the changes in this release, only used for branch releases
```

Comment:
I was a bit confused by this phrasing until you explained it on the team meeting. Maybe expand this a bit to say:
"A text message summarizing the changes in this release. Unused for standard major/minor/patch releases, but used for branch releases."

I'm not sure that's great, but just something clarifying that it doesn't have an expected use in "normal" release flows.